playbook.yml

```diff
@@ -0,0 +1,168 @@
+- hosts: hub
+  tasks:
+    - name: Enable Ansible PPA repo
```

Comment:
Honestly I'd recommend just using pip to install ansible, but that might be the python guy talking.

Reply:
+1 I no longer futz with apt to install Ansible, just go through pip. Might just have been my own bad experience getting older ansible versions via apt because I wasn't using the ansible ppa.